config/routes.rb

```diff
@@ -0,0 +1,19 @@
+Rails.application.routes.draw do
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
+  # root "articles#index"
+  devise_for :users
+  root 'public_recipes#index'
+  resources :users do
+    resources :recipes, except: %i[update, edit] do
+      get '/general_shopping_list', to: 'shopping_list#index', as: 'general_shopping_list'
+      resources :recipe_foods, only: [:new, :create, :destroy, :update, :edit]
```

Comment:
- [ ] Please as you can see in this given [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/general_shopping_list.png?raw=true) the list of the general shopping list should match the URL `/general_shopping_list` but currently have this URL `recipes/:id/general_shopping_list` :blush:  as you can see below :point_down: 

  ![image](https://user-images.githubusercontent.com/92303891/226051576-48434171-9087-4fe2-a9bf-65a3d7afbb01.png)